h2o-algos/src/main/java/hex/glm/GLMTask.java

```diff
@@ -2036,12 +2043,17 @@ protected void processRow(Row r) {
         w = _glmw.w;
       }
       double eta = r.innerProduct(_betaNew) + _sparseOffsetNew;
-//      double mu = _parms.linkInv(eta);
-      _sumsqe += w*(eta - z)*(eta - z);
+      double mu = _glmf.linkInv(eta);
+      _sumsqe += _glmf._family.equals(Family.tweedie)?
+              r.weight*((r.response(0)-mu)*(r.response(0)-mu))/Math.pow(mu, _glmf._var_power):
+              w*(eta - z)*(eta - z);
+
```

Comment:
implementing the dispersion factor calculation for tweedie according to paper http://people.stat.sfu.ca/~raltman/stat402/402L25.pdf